packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-node-julia.vue

```diff
@@ -126,11 +132,11 @@ import InputNumber from 'primevue/inputnumber';
 import InputText from 'primevue/inputtext';
 import Dropdown from 'primevue/dropdown';
 import { Poller, PollerState } from '@/api/api';
-import { EventSourceManager } from '@/api/event-source-manager';
 import TeraSimulateChart from '@/workflow/tera-simulate-chart.vue';
 import TeraProgressBar from '@/workflow/tera-progress-bar.vue';
 import { getTimespan } from '@/workflow/util';
 import { logger } from '@/utils/logger';
+import { subscribe, unsubscribe } from '@/services/ClientEventService';
```

Comment:
```suggestion
import * as ClientEventService from '@/services/ClientEventService';
```

Better to then call `ClientEventService.subscribe()`